src/pipeline/common/PullRequest.groovy

```diff
@@ -0,0 +1,61 @@
+package pipeline.common
+
+def addCommentsToPR(String library_dir, String manifest_path, String public_jenkins_url){
```

Comment:
can you add a simple comment to tell what is the comment ? cry to tears....
otherwise, people may think "which parameter is the comment's content ? "

//this comment will be added when Jenkins test fails.
//-----------------------------------------
//  JenkinsRHD commented on xx-xx
//
//   BUILD on-http #302 : FAILURE
//-------------------------------------------

Reply:
Thanks, fixed